src/views/VerifyBackupPhrase/VerifyBackupPhrase.tsx

```diff
@@ -31,16 +31,22 @@ export function VerifyBackupPhrase({ backupPhrase }: Props): JSX.Element {
   const error = !wordsAreInOrder && t('view_VerifyBackupPhrase_error');
 
   const selectWord = useCallback(
-    (event) => {
-      const selectedIndex = parseInt(event.target.value, 10);
+    (event: MouseEvent) => {
+      const selectedIndex = parseInt(
+        (event.target as HTMLButtonElement).value,
+        10,
+      );
```

Comment:
Would that work?
```suggestion
    (event: MouseEvent<HTMLButtonElement>) => {
      const selectedIndex = parseInt(event.target.value, 10);
```

Reply:
After some looking into it, I realized that the type parameter specifies the type of `currentTarget` which we should use.